src/components/open-media-button.js

```diff
@@ -1,9 +1,40 @@
 import { isHubsSceneUrl, isHubsRoomUrl } from "../utils/media-utils";
 import { guessContentType } from "../utils/media-utils";
 
+AFRAME.registerComponent("window-open-button", {
```

Comment:
what's the motivation for breaking this out vs just incorporating the necessary logic into `open-media-button`?

Reply:
We use this to open links to anything (including scenes and rooms) in another tab, rather than to change the scene or navigate to a room. They have two things in common: 
- Both want to get the `src` from the `media-loader`
- Both do something on click
But besides that they serve different purposes.